zephyr/CMakeLists.txt

```diff
@@ -88,6 +88,10 @@ target_include_directories(SOF INTERFACE ../zephyr/include)
 target_include_directories(SOF INTERFACE ${SOF_SRC_PATH}/include)
 target_include_directories(SOF INTERFACE ${SOF_SRC_PATH}/arch/${ARCH}/include)
 
+# TODO: The xtos-wrapper include PATH will eventually become optional based on the
+# RTOS selection. It's manadatory atm until the native Zephyr integration is ready.
+target_include_directories(SOF INTERFACE ${SOF_SRC_PATH}/arch/xtos-wrapper/include)
```

Comment:
hm, is this needed even if `CONFIG_ZEPHYR_NATIVE_DRIVERS` is used? If not, maybe make it conditional on this, if yes - maybe keep the headers, that Zephyr builds still need at their current locations and only move them over one by one as we stop using them in Zephyr?

Reply:
yep, that the plan and why it's a TODO. Its still WIP atm.